server/src/main/java/org/elasticsearch/ElasticsearchException.java

```diff
@@ -1027,7 +1027,9 @@ public String toString() {
                 org.elasticsearch.common.xcontent.UnknownNamedObjectException::new, 148, Version.V_5_2_0),
         TOO_MANY_BUCKETS_EXCEPTION(MultiBucketConsumerService.TooManyBucketsException.class,
                                    MultiBucketConsumerService.TooManyBucketsException::new, 149,
-            Version.V_6_2_0);
+            Version.V_6_2_0),
+        SNAPSHOT_IN_PROGRESS_EXCEPTION(org.elasticsearch.snapshots.SnapshotInProgressException.class,
```

Comment:
Can you add a comment for number 150 other comments for missing numbers in this list? It's nice to know that the increment is intentional.

Reply:
yes!